src/modules/Dashboard/features/Respondents/Popups/EditRespondentPopup/EditRespondentPopup.tsx

```diff
@@ -86,39 +87,70 @@ export const EditRespondentPopup = ({
       onClose={onCloseHandler}
```

Comment:
The table shouldn't refresh after cancelling the edit action by hitting close (I noticed this omission in a couple other popups I worked on):
```suggestion
      onClose={() => onCloseHandler(false)}
```